docs/site/Importing-LB3-models.md

```diff
@@ -0,0 +1,79 @@
+---
+lang: en
+title: 'Importing models from LoopBack 3 projects'
+keywords: LoopBack 4.0, Migration
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Importing-LB3-models.html
+---
+
+## Synopsis
+
+To simplify migration from LoopBack 3, LoopBack 4 provides a CLI tool to import
+LoopBack 3 models into your LoopBack 4 project.
+
+{% include warning.html content="
+This command is experimental and not feature-complete yet.
+See the list of known limitations below.
+" %}
+
+## Overview
+
+A single model can be imported by running `lb4 import-lb3-model` command.
```

Comment:
Is it assumed that the command for multiple models is `lb4 import-lb3-models`?

Reply:
Looks like there is no `lb4 import-lb3-models`.

The statement "A single model can be imported by running `lb4 import-lb3-model` command.", makes it sound like there may be something for multiple models.

So do we have to execute `lb4 import-lb3-model` as many times as there are model that one wants to import?

Why not prompt, "Do you want to import <model>?" one after another?

Reply:
I think the agreement and implementation allow multiple models to be imported in one run. Let's rename `import-lb3-model` to `import-lb3-models` across the board.

Reply:
> I think the agreement and implementation allow multiple models to be imported in one run.

+1

My very first draft implementation was for a single model only. I forgot to update the documentation page after I reworked the code 🙈 

> Let's rename `import-lb3-model` to `import-lb3-models` across the board.

Will do.